packages/phpstan-rules/tests/Rules/NoFuncCallInMethodCallRule/Fixture/SkipNamespacedFunction.php

```diff
@@ -4,16 +4,27 @@
 
 namespace Symplify\PHPStanRules\Tests\Rules\NoFuncCallInMethodCallRule\Fixture;
 
+use Symfony\Component\DependencyInjection\Loader\Configurator\ReferenceConfigurator;
 use function Symfony\Component\DependencyInjection\Loader\Configurator\ref;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
 
 final class SkipNamespacedFunction
 {
     public function something()
     {
-        $this->process(ref('reference'));
+        $this->process($this->getReferenceConfigurator());
```

Comment:
This breaks the test, as the test is checking function in method call:

```php
$this->methodCall(some_function());
```

Reply:
What I meant by adding custom namespaced function is creating a new file in source:


```php
namespace Symplify\PHPStanRules\Tests\Rules\NoFuncCallInMethodCallRule\Source;

function some_function()
{
}
```

← The function file must be included in `composer.json` required-dev > files.

<br>

And using it here instead:

```diff
-$this->process(ref('reference'));
+$this->process(some_function('reference'));
```

Reply:
Yeah, sorry. It looks like you didn't check my last commit, I realized that, but forgot to add it to composer.json. By require-dev you mean autoload-dev -> files, right?

Reply:
> By require-dev you mean autoload-dev -> files, right?

Yes yes :+1: